CSparse.Tests/Double/SparseMatrixTest.cs

```diff
@@ -229,25 +229,35 @@ public void TestMatrixMultiply(int rows, int columns)
             CollectionAssert.AreEqual(data.AmBT.Values, actual.Values);
         }
 
-        [TestCase(2, 2)]
-        [TestCase(2, 3)]
-        public void TestMatrixParallelMultiply(int rows, int columns)
+        [Test]
+        public void TestMatrixParallelMultiply()
         {
-            var data = MatrixHelper.LoadSparse(rows, columns);
-
-            var A = data.A;
-            var B = data.B;
-
-            var AT = data.AT;
-            var BT = data.BT;
-
-            var actual = AT.ParallelMultiply(B);
-
-            CollectionAssert.AreEqual(data.ATmB.Values, actual.Values);
-
-            actual = A.ParallelMultiply(BT);
-
-            CollectionAssert.AreEqual(data.AmBT.Values, actual.Values);
+            var data = ResourceLoader.Get<double>("general-40x40.mat");
+            var acs = new CSparse.Storage.CoordinateStorage<double>(40, 400, 40 * 400);

```

Comment:
`acs` and `bcs` should be initialized in a `foreach`-loop using `data.EnumerateIndexed()`:
```c#
foreach (var item in data.EnumerateIndexed())
{
    int i = item.Item1;
    int j = item.Item2;

    for (var k = 0; k < 10; k++)
    {
        acs.At(i, j + 40 * k, item.Item3);
        bcs.At(i + 40 * k, j, item.Item3);
    }
}
```

Reply:
Did that. See next commit.